### PR TITLE
Set EnsureWinRT property in configuration system.

### PR DESCRIPTION
**Pull request by weshaggard**
EnsureWinRT is set based on netcore50/uap target frameworks.

Removed mscorlib.WinRt-Facade project which is no longer needed.

cc @joperezr @mellinoe @JeremyKuhne

### CODE REVIEW

File: src/System.IO.IsolatedStorage/src/System.IO.IsolatedStorage.csproj [hunk 1]
```diff
@@ -4,17 +4,12 @@
   <PropertyGroup>
     <AssemblyName>System.IO.IsolatedStorage</AssemblyName>
     <ProjectGuid>{FAF5D1E4-BA43-4663-8429-C069066D75CB}</ProjectGuid>
-    <EnableWinRT Condition="'$(TargetGroup)' == 'uap101'">true</EnableWinRT>
```
Comment (joperezr) [inline src/System.IO.IsolatedStorage/src/System.IO.IsolatedStorage.csproj]: should we set this to true for TargetGroup = uap? Or is that broken as well?

Comment (joperezr) [inline src/System.IO.IsolatedStorage/src/System.IO.IsolatedStorage.csproj]: oh nvm, I just saw that you did this for the whole targetGroup

Comment (weshaggard) [inline src/System.IO.IsolatedStorage/src/System.IO.IsolatedStorage.csproj]: It is set to true now in our configuration.props we generate.

**Comment by jkotas** [comment]
> EnsureWinRT is set based on netcore50/uap target frameworks.

Should this say EnableWinRT ... ?

**Comment by weshaggard** [comment]
>Should this say EnableWinRT ... ?

Yes thanks for the catch I will update the commit message.

**Pull request merged**